docs/zkapps/o1js/keccak.mdx

```diff
@@ -108,7 +108,7 @@ console.log(hash.toHex());
 //69070dda01975c8c120c3aada1b282394e7f032fa9cf32f4cb2259a0897dfc04
 ```
 
-See the o1js repository for an [example](https://github.com/o1-labs/o1js/tree/main/src/examples/zkapps/hashing) of how to use Keccak. 
+See the o1js repository for a [hashing example](https://github.com/o1-labs/o1js/tree/main/src/examples/zkapps/hashing) that uses SHA-256 and Keccak.
```

Comment:
@Trivo25  is calling out SHA-256 and Keccak too detailed?

Reply:
Nope, I think it reads well and makes sense!